libsplinter/src/biome/rest_api/resources/credentials.rs

```diff
@@ -14,7 +14,7 @@
 
 //! Defines credentials used to register and authenticate users.
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Serialize)]
```

Comment:
This shouldn't actually be sent back as is - we should minimize the number of times the hashed password should be sent over the wire.

Reply:
Addressed by creating a `NewUser` object which is now returned by the /register endpoint